tests/test_vms_platform.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+from qubesadmin import Qubes
+
+
+SUPPORTED_PLATFORMS = [
+    "Fedora 25 (Twenty Five)",
```

Comment:
One question to @joshuathayer, do we want to update to Fedora 26 for the vms?

Reply:
Yes, I think we should. There's an issue open for it (#48). What do you think?

Reply:
Yes, #48 is tracking the migration to F26—the tests are just an outline for tests for that migration. These tests pass, but when we resolve #48, we'll need to update these same tests, since they _should_ start failing under F26.